### DISCUSSION TITLE
termux python problem

### DISCUSSION THREAD

**Discussion by montassarayed** [termux/termux-packages#8201] [Q&A] [answered]
i face this problem in termux any tool in python cant work
![Screenshot_20211210_231126_com termux](https://user-images.githubusercontent.com/40012846/145647948-cbb08ceb-3f27-4123-9c88-f4c6d9cbd326.jpg)

**Comment by agnostic-apollo** [comment, marked as answer]
You are running an apparent `python` shell script with `bash` shell. Those a different shells. Use `python ASU.py`.

**Reply by montassarayed** [reply]
![Screenshot_20211211_181959_com termux](https://user-images.githubusercontent.com/40012846/145685648-b800fbe4-49fb-4e07-be2f-71b5c8966259.jpg)

**Reply by agnostic-apollo** [reply]
That is an issue with `ASU` hacking tool you are using and not a termux issue. Open issue in respective repo. Moreover, asking for support related to hacking tools is banned on our support forums.

https://wiki.termux.com/wiki/Hacking

**Comment by montassarayed** [comment]
another problem
